pkg/xds/topology/outbound.go

```diff
@@ -64,6 +70,24 @@ func BuildEndpointMap(destinations core_xds.DestinationMap, dataplanes []*mesh_c
 				})
 			}
 		}
+
+		for _, externalService := range externalServices {
```

Comment:
Correct me if I'm wrong, but look like this loop has to be placed out of the loop in line 29